Sources/SwiftUIPager/Helpers/PositionAlignment.swift

```diff
@@ -29,15 +34,15 @@ public enum PositionAlignment {
         switch self {
         case .center:
             return 0
-        case .end(let insets), .start(let insets):
+        case .end(let insets), .start(let insets), .justified(let insets):
```

Comment:
This is the only thing I wasn't sure about. As far as I can see `alignment.inset` is only used [here](https://github.com/fermoya/SwiftUIPager/blob/master/Sources/SwiftUIPager/Pager%2BHelper.swift#L178) and I'm not sure if the value works out as it was intended here.